releasenotes/notes/add-release-notes-generation.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: release-notes/v1
+
+# This YAML file describes the format for specifying a release notes entry for Istio.
+# This should be filled in for all user facing changes.
+
+# kind describes the type of change that this represents. 
+# Valid Values are:
+# - bug-fix -- Used to specify that this change represents a bug fix. 
+# - security-fix -- Used to specify that this change represents a security fix. 
+# - feature -- Used to specify a new feature that has been added. 
+# - test -- Used to describe additional testing added. This file is optional for
+#   tests, but included for completeness.
+kind: feature
+
+# area describes the area that this change affects. 
```

Comment:
can we remove these comments?

Reply:
Removed the comments. This content is already in the template and readme as well.